solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -55,15 +55,9 @@ tag_name            = "cortx-multinode"
 ```
 terraform validate && terraform apply -var-file user.tfvars --auto-approve
 ```
-- Execute the following commands to find the Public and Private ip addresses from local host by setting as environment variable.
-
-**Public ip address:**
-```
-PUBLIC_IP=`terraform show -json terraform.tfstate | jq .values.outputs.aws_instance_public_ip_addr.value 2>&1 | tee ip_public.txt | tr -d '",[]' | sed '/^$/d'`
+- Execute the following command as environment variable
```

Comment:
This should be `Execute the following command to store AWS instance public IP addresses in PUBLIC_IP variable`

Reply:
fixed